frontend/src/components/Review/Review.tsx

```diff
@@ -99,14 +108,23 @@ const ReviewComponent = ({
     photoRowStyle,
     bottomborder,
     heartSpacing,
+    landlordIndicator,
   } = useStyles();
   const [expanded, setExpanded] = useState(false);
   const [expandedText, setExpandedText] = useState(false);
+  const [landlord, setLandlord] = useState<Landlord>();
 
   const handleExpandClick = () => {
     setExpanded(!expanded);
   };
 
+  //Retreieving landlord data
+  useEffect(() => {
+    get<Landlord>(`/api/landlord/${review.landlordId}`, {
+      callback: setLandlord,
+    });
+  }, [review]);
+
```

Comment:
I think this API call isn't actually necessary. The parent component already should have made this API call and acquired the data already (since it's the landlord page), and since you only need the name of the landlord, it would make sense to just pass the name of it as a prop to this component, especially since this component is used in other places where you don't need the landlord name.

I would suggest adding an optional prop to this component that passes in the landlord name, and in 188 replace the `landlord?.name` with this prop, and maybe change the condition so it relies on the landlord name being not null instead.